arch/arm/configs/bcm2709_defconfig

```diff
@@ -874,6 +874,7 @@ CONFIG_VIDEO_EM28XX_V4L2=m
 CONFIG_VIDEO_EM28XX_ALSA=m
 CONFIG_VIDEO_EM28XX_DVB=m
 CONFIG_V4L_PLATFORM_DRIVERS=y
+CONFIG_VIDEO_BCM2835_UNICAM=m
```

Comment:
Github is howing this defconfig change as part of "media: uapi: v4l2-core: Add sensor ancillary data V4L2 foucc type."

Reply:
Oops, that was my fault with the rebasing.

Fixed.